terminus-ui/table/src/row.ts

```diff
@@ -11,6 +11,7 @@ import {
   Directive,
   ViewEncapsulation,
 } from '@angular/core';
+import { TsTableComponent } from './table.component';
```

Comment:
was wondering how this could have worked without the import,.. then found out it lives in the comments..